content/en/docs/tasks/security/authorization/authz-ingress/index.md

```diff
@@ -31,40 +31,123 @@ through the Istio ingress gateway with this command:
-    to define the `INGRESS_HOST` and `INGRESS_PORT` environment variables.
-
 * Verify that the `httpbin` workload and ingress gateway are working as expected using this command:
 
     {{< text bash >}}
     $ curl "$INGRESS_HOST":"$INGRESS_PORT"/headers -s -o /dev/null -w "%{http_code}\n"
     200
     {{< /text >}}
 
-* Verify the output of the following command to ensure the ingress gateway receives
-the original client source IP address, which will be used in the authorization policy:
-
-    {{< text bash >}}
-    $ CLIENT_IP=$(curl "$INGRESS_HOST":"$INGRESS_PORT"/ip -s | grep "origin" | cut -d'"' -f 4) && echo "$CLIENT_IP"
-    105.133.10.12
-    {{< /text >}}
-
 {{< warning >}}
 If you don’t see the expected output, retry after a few seconds.
 Caching and propagation overhead can cause a delay.
 {{< /warning >}}
 
+## Getting traffic into Kubernetes and Istio
+
+* All methods of getting traffic into Kubernetes involve opening a port on all worker nodes.  The main features that accomplish this are the `NodePort` service and the `LoadBalancer` service.  Even the Kubernetes `Ingress` resource must be backed by an Ingress controller that will create either a `NodePort` or a `LoadBalancer` service.
+
+* A `NodePort` just opens up a port in the range 30000-32767 on each worker node and uses a label selector to identify which Pods to send the traffic to.  You have to manually create some kind of load balancer in front of your worker nodes or use Round-Robin DNS.
+
+* A `LoadBalancer` is just like a `NodePort`, except it also creates an environment specific external load balancer to handle distributing traffic to the worker nodes.  For example, in AWS EKS, the `LoadBalancer` service will create an ELB with your worker nodes as targets.  If your Kubernetes environment does not have a `LoadBalancer` implementation, then it will just behave like a `NodePort`.  An Istio ingress gateway creates a `LoadBalancer` service.
+
+* What if the Pod that is handling traffic from the `NodePort` or `LoadBalancer` isn't running on the worker node that received the traffic?  Kubernetes has its own internal proxy called kube-proxy that receives the packets and forwards them to the correct node.
+
+## Source IP address of the original client
+
+* If a packet goes through an external load balancer and/or kube-proxy, then the original source IP address of the client is lost.  There are a few ways to handle this if you need the original client IP for logging or security purposes:
+
+      1. If you are using an HTTP/HTTPS external load balancer, it can put the original client IP address in the X-Forwarded-For header.  Istio can extract the client IP address from this header with some configuration.  See [Configuring Gateway Network Topology](/docs/ops/configuration/traffic-management/network-topologies/). Quick example if using a single load balancer in front of Kubernetes:
```

Comment:
would it be better if we put the 3 options in a tab?

Reply:
This one is better left as a numbered list, IMO.

Reply:
I made it a tab before I saw Frank's comment.  Let me know whether you like it this way or the other way.  I can change it back if need be.